miio/tests/test_vacuums.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022
+# Author: 2pirko
```

Comment:
There are no copyright statements anywhere in the code base, so if it's fine for you I'd remove this. The git history will keep track of contributions, which makes it much easier than having the potential need to update these in the future.